fvm/fvm_test.go

```diff
@@ -1287,6 +1287,7 @@ func TestSignatureVerification(t *testing.T) {
 			)
 
 			t.Run(fmt.Sprintf("%s %s", signatureAlgorithm.name, hashAlgorithm.name), func(t *testing.T) {
+				t.Skip("temporarily disabled")
```

Comment:
Instead of skipping for all pairs, can we only skip for the excluded case(s) above?

Reply:
This skip is no longer needed, since the change in VerifySignature was reverted.